### DISCUSSION TITLE
SPI reports 2 data race safety errors but I can’t find them

### DISCUSSION THREAD

**Discussion by Frizlab** [SwiftPackageIndex/SwiftPackageIndex-Server#3564] [Q&A] [answered]
On [this page](https://swiftpackageindex.com/xcode-actions/swift-signal-handling/builds) I’m told there are two data race safety issues for version `1.2.0-beta.1` (e.g. [this build](https://swiftpackageindex.com/builds/FD537BDE-81F4-4081-A156-0204B7999136) but all builds for this version have the same number of issues), but I can’t find the issue anywhere in the logs, neither when building locally or in the logs on SPI.

Can you help me?

**Comment by finestructure** [comment, marked as answer]
It looks like this is another case of a false negative being reported by the Swift compiler. We're tracking this [in issue 3233](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/3233) but unfortunately there doesn't seem to be anything we can do at the moment to fix this.

(I can confirm that the warning reporting does 2 warnings in the output file when the build doesn't actually show any warnings.)

**Comment by daveverwer** [comment]
We also don't do a good job of letting people know that this is an issue, so I have added a note to the section of the FAQ linked under the data race notice.

See #3565

**Reply by Frizlab** [reply]
Awesome. Thanks!
